### PR TITLE
[Cleanup]: Fix Android Studio Warnings {regarding Constant Conditions} #13282

### PR DESCRIPTION
**Pull request by xenonnn4w**
## Purpose / Description
Getting Familiar with Project and to sort out weak warnings .#13282

## Fixes
* https://github.com/ankidroid/Anki-Android/issues/13282
1. Changed Value of Var to Val 
2. Replaced the `if (!isValidScopeName) return isValidScopeName` with `if (!isValidScopeName) return false` as it was always false .

## Approach
Removal of warnings, to maintain the clean code practice

## Learning (optional, can help others)
Was not able to inspect code with `Code -> Inspect Code`  had to do as follows :
image

### CODE REVIEW

File: AnkiDroid/src/main/java/com/ichi2/anki/jsaddons/NpmUtils.kt [hunk 1]
```diff
@@ -83,10 +83,7 @@ object NpmUtils {
             val arr = name.split('/')
             val scope = arr[0].removePrefix("@")
             val isValidScopeName = validateName(scope)
-
-            if (!isValidScopeName) {
-                return isValidScopeName
-            }
+            if (!isValidScopeName) return isValidScopeName
```
Comment (david-allison) [inline AnkiDroid/src/main/java/com/ichi2/anki/jsaddons/NpmUtils.kt:86]: The proposed fix didn't fix the warning

```suggestion
            if (!isValidScopeName) return false
```

**Review by david-allison — CHANGES_REQUESTED** [review]
One quick fix, then looks good to go!

File: AnkiDroid/src/main/java/com/ichi2/anki/jsaddons/NpmUtils.kt (hunk 1 above)
Comment (xenonnn4w) [reply AnkiDroid/src/main/java/com/ichi2/anki/jsaddons/NpmUtils.kt]: Done .

**Review by david-allison — APPROVED** [review]
Perfect, thanks!

**Pull request merged**